mne/io/edf/edf.py

```diff
@@ -506,6 +506,20 @@ def _get_info(fname, stim_channel, eog, misc, exclude, preload):
     return info, edf_info, orig_units
 
 
+def _parse_prefilter_string(prefiltering: str):
```

Comment:
Do we use type annotations yet?

Reply:
No not yet. We probably shouldn't add them here until we discuss how to do it properly and project-wide